ConsistencyManager/ConsistencyManager.swift

```diff
@@ -632,6 +672,13 @@ open class ConsistencyManager {
                     listener.modelUpdated(newModel, updates: modelUpdates, context: context)
                 }
             }
+            self.modelUpdatesListeners.forEach { listener in
+                listener?.consistencyManager(
```

Comment:
Consider renaming variable as it is the same as the closure property name above.